src/runtime/language.c

```diff
@@ -0,0 +1,9 @@
+#include "runtime/parser.h"
```

Comment:
I think this should be `#include "tree_sitter/parser.h"` (the header in the `include` directory), since `struct TSLanguage` is defined there. Sorry, it's a bit confusing to have two files called `parser.h`. I think this happens to work because `tree_sitter/parser.h` gets transitively included through a couple of steps.